src/components/Nav.js

```diff
@@ -1,71 +1,39 @@
 import React, { Component } from 'react';
-import AppBar from 'material-ui/AppBar';
-import Drawer from 'material-ui/Drawer';
-import MenuItem from 'material-ui/MenuItem';
 import { Link } from 'react-router-dom';
 
 class Nav extends Component {
-  constructor(props) {
-    super(props);
 
-    this.state = {
-      drawerOpen: false,
-    };
-
-    this.handleDrawerToggle = this.handleDrawerToggle.bind(this);
-  }
-  handleDrawerToggle() {
-    this.setState({
-      drawerOpen: !this.state.drawerOpen
-    });
-  }
   render () {
     return (
       <div>
-        <AppBar
-          title={
-            <Link style={{
-              color: '#ffffff',
-              textDecoration: 'none'
-            }} to="/">
-              Adopt-a-Family
+        <div>
```

Comment:
We have 2 empty `<div/>`'s in a row w/o any class names. Do we need these?

Reply:
![image](https://user-images.githubusercontent.com/12107969/32132459-d92da158-bb78-11e7-8c14-8f466bfbcb06.png)